DesktopEdge/Views/ItemRenderers/MenuEditItem.xaml

```diff
@@ -5,15 +5,15 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              xmlns:local="clr-namespace:ZitiDesktopEdge"
              mc:Ignorable="d" MaxWidth="360"
-             d:DesignHeight="31" Height="31" HorizontalAlignment="Stretch" d:DesignWidth="350" Cursor="Hand">
+             d:DesignHeight="31" Height="31" HorizontalAlignment="Stretch" d:DesignWidth="350">
     <StackPanel Orientation="Vertical" Height="31">
         <Grid Height="30">
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="Auto"/>
                 <ColumnDefinition Width="1*"/>
             </Grid.ColumnDefinitions>
             <TextBox Name="MainLabel" Background="White" Height="30" FontWeight="Bold" IsReadOnly="True" HorizontalAlignment="Stretch" BorderThickness="0" ClipToBounds="True" Padding="0,6,3,0" Grid.Column="0" VerticalAlignment="Center" FontSize="12" FontFamily="pack://application:,,,/Assets/Fonts/#Open Sans" Foreground="Black"></TextBox>
-            <TextBox Name="MainEdit" Background="White" IsReadOnly="True" HorizontalAlignment="Stretch" Height="30" Grid.Column="1" BorderThickness="0" TextAlignment="Right" Padding="0,6,3,0" FontSize="12" FontFamily="pack://application:,,,/Assets/Fonts/#Open Sans" ></TextBox>
+            <TextBox Name="MainEdit" Cursor="ScrollWE" Background="White" IsReadOnly="True" HorizontalAlignment="Stretch" Height="30" Grid.Column="1" BorderThickness="0" TextAlignment="Right" Padding="0,6,3,0" FontSize="12" FontFamily="pack://application:,,,/Assets/Fonts/#Open Sans" ></TextBox>
```

Comment:
Explicitly assigning the cursor "ScrollWE" is a visual cue that is better than the hand cursor.  This cursor indicates to the user that the field is actually longer than what is potentially displayed.  The cue helps the user understand that they can click+drag across the text box to see/copy more of the text in that text box.
![image](https://github.com/user-attachments/assets/679fad86-9b87-410d-91d2-ebee41e5b508)